Mathport/Syntax/Parse.lean

```diff
@@ -1023,10 +1027,10 @@ def RawTacticInvocation.build
 end
 
 def RawAST3.build : RawAST3 → (invocs :_:= true) → Except String (AST3 × Array AST3.TacticInvocation)
-| ⟨ast, file, level, expr, tactics, states⟩, invocs => do
+| ⟨ast, file, level, expr, tactics, states, comments⟩, invocs => do
   let level := buildLevels level
   let expr := buildExprs level expr
-  M.run ast expr $ do
+  M.run ast comments expr $ do
     let (ast, tacs) ← getAST file
```

Comment:
```suggestion
  M.run ast expr $ do
    let (ast, tacs) ← getAST file comments
```
As far as I can see there is no need to have the comments in the `Context` for the parser

Reply:
Good catch, I'll push the change once CI is finished.